lib/node_modules/@stdlib/stats/base/dsnanmean/lib/ndarray.js

```diff
@@ -30,22 +30,17 @@ var dsnanmeanpn = require( '@stdlib/stats/base/dsnanmeanpn' ).ndarray;
 *
 * @param {PositiveInteger} N - number of indexed elements
 * @param {Float32Array} x - input array
-* @param {integer} stride - stride length
-* @param {NonNegativeInteger} offset - starting index
+* @param {integer} strideX - stride length
+* @param {NonNegativeInteger} offsetX - starting index
 * @returns {number} arithmetic mean
 *
 * @example
 * var Float32Array = require( '@stdlib/array/float32' );
-* var floor = require( '@stdlib/math/base/special/floor' );
 *
 * var x = new Float32Array( [ 2.0, 1.0, 2.0, -2.0, -2.0, 2.0, 3.0, 4.0, NaN ] );
-* var N = floor( x.length / 2 );
-*
-* var v = dsnanmean( N, x, 2, 1 );
-* // returns 1.25
```

Comment:
Why was this removed? Don't remove inconvenient code (e.g., if lint errors are happening, fix the lint errors; don't just delete it).